ipyplotly/.gitignore

```diff
@@ -0,0 +1,27 @@
+*.egg-info/
```

Comment:
@Kully - It looks like you committed a lot of extra stuff, like this `ipyplotly` folder. Was that intentional? If not, could you rebase and open up a new PR?

Reply:
I highly recommend using a git client like GitHub desktop https://desktop.github.com/ to make your commits - it makes a lot easier to view what you will commit before committing it, as well as organizing your commits to only include certain files and line numbers

Reply:
It was intentional. `ipyplotly`can't be located without the folder in ploty.py